hms-app/src/common/constants.ts

```diff
@@ -1 +1,3 @@
 export const NULL_DATE = new Date('1970-01-01')
+export const HeaderActiveColor = 'lightblue'
```

Comment:
so I think there is already a `styles` file that this wold fit more logically in

Reply:
also the capitalization is probably wrong, constants should be ALL_CAPS or possibly camalCase